examples/image_database/write_example.cc

```diff
@@ -0,0 +1,59 @@
+// BoB robotics includes
+#include "common/path.h"
+#include "navigation/image_database.h"
+#include "video/randominput.h"
+
+// Third-party includes
+#include "third_party/units.h"
+
+using namespace BoBRobotics;
+using namespace units::length;
+using namespace units::literals;
+
+int bobMain(int, char **argv)
+{
+    // Make a new image database; make folder with random name in current folder
+    const auto programPath = filesystem::path{ argv[0] }.parent_path();
```

Comment:
This is more an FYI than anything I think should be fixed here but,  ``argv[0]`` is not very good for this - it's never an absolute path. Sadly the solutions are messy - https://stackoverflow.com/questions/933850/how-do-i-find-the-location-of-the-executable-in-c - different for mac, linux and windows - yey